src/amo/pages/Guides/index.js

```diff
@@ -173,21 +170,18 @@ export class GuidesBase extends React.Component<InternalProps> {
   }
 }
 
-export const mapStateToProps = (state: AppState) => {
-  const { clientApp, lang } = state.api;
-
+export function mapDispatchToProps(dispatch: DispatchFunc) {
   return {
-    clientApp,
-    lang,
+    dispatch,
   };
-};
+}
 
 export const extractId = (ownProps: InternalProps) => {
   return ownProps.match.params.slug;
 };
 
 const Guides: React.ComponentType<Props> = compose(
-  connect(mapStateToProps),
+  connect(mapDispatchToProps),
```

Comment:
So, I don't know how this is supposed to work because the first expected argument of `connect` should be `mapStateToProps`. Given that only `dispatch` needs to be injected, you can completely remove `mapDispatchToProps` here and above so that `connect()` is empty and `dispatch` is automatically injected.

Reply:
oh?! ok I needed dispatch but I didn't realize connect() could be empty..I'll update